oggm/utils/_downloads.py

```diff
@@ -981,7 +981,7 @@ def _download_dem3_viewpano_unlocked(zone):
         ifile = 'http://viewfinderpanoramas.org/dem3/' + zone + 'v2.zip'
     elif zone in DEM3REG.keys():
         # We prepared these files as tif already
-        ifile = ('https://cluster.klima.uni-bremen.de/~oggm/DEM/'
+        ifile = ('https://cluster.klima.uni-bremen.de/~oggm/dem/'
```

Comment:
How was it possible to create the RGI-TOPO files? Is there still the old path on cluster maybe?

Reply:
just checked: I did the RGI topo runs 2-3 days after the paths changed in #981 
But by chance I still manually installed the commit 1 before the change in singularity. And the old path with your name is still a symlink to the new path :-)